src/Components/RequestGeneratorBuilder.js

```diff
@@ -201,7 +196,10 @@ export default class RequestGeneratorBuilder extends preact.Component {
             // requests anyway in that they are either also to tracking companies or those companies at least
             // identify the user by the same details (i.e. cookie IDs, device IDs, etc.)
             // I couldn't come up with a better name, so we'll just leave them as tracking requests, I guess…
-            prev.request.is_tracking_request = ['access-tracking'].includes(template_file);
+            // TODO: Get rid of the `.txt` replacement.
+            prev.request.is_tracking_request = ['access-tracking'].includes(
```

Comment:
If we're doing `'custom-' + this.state.request.type + '-template'` anyway, we might as well plan for the future and include `erasure-tracking` and `rectification-tracking` here.

I know this was *technically* the same in the old code, as well but it became more apparent now. :D